modules/sdk/project-templates/project-templates-social-bookmark/src/main/resources/archetype-resources/src/main/java/social/bookmark/__className__SocialBookmark.java

```diff
@@ -31,8 +31,8 @@
 
 	@Override
 	public String getName(Locale locale) {
-		ResourceBundle resourceBundle =
-			_resourceBundleLoader.loadResourceBundle(locale);
+		ResourceBundle resourceBundle = ResourceBundleUtil.getBundle(
```

Comment:
Hello @marianoalvarosaiz   Thank you for this update.  Does this only work in 7.1 or 7.2?  Or has it been backported to 7.0?  

If there is a difference, between 7.0 and 7.1 (7.2) can you look at the other templates and see how we are using #velocity tags to branch the template code that is generated.  E.g. 

```
#if ${liferayVersion.startsWith("7.1")}
...
#else
...
#end
```

Let me know if you have any further questions.  You can contact @lawrence-lee in the US if you need more information of how to test your changes (also we have Unit tests that will need to pass over in ProjectTemplateTests) class.  So if you could help out with adding a test, it would be most appreciated!

Reply:
Hi @gamerson,
 the changes in ResourceBundleUtil have already been backported in LPS-83494 to 7.0 on. Anyway, ResourceBundleUtil.getBundle is already used in __className__PanelCategory.java, __className__PortletConfigurationIcon.java, __className__PortletToolbarContributor.java, ... so it's already working. Is the usage of ResourceBundleLoader as reference which has been discouraged in favour of the Util class since declaring it without a greedy/dynamic policy implies the impossibility to override ResourceBundle.

I'm going to sent a new pull with a test to check that no private ResourceBundleLoader, protected ResourceBundleLoader attribute is declared.

Kind regards.